crates/stdarch-verify/tests/arm.rs

```diff
@@ -331,6 +331,46 @@ fn verify_all_signatures() {
                 "_cls_u32",
                 "_cls_u64",
                 "_prefetch",
+                "vsli_n_s8",
+                "vsliq_n_s8",
+                "vsli_n_s16",
+                "vsliq_n_s16",
+                "vsli_n_s32",
+                "vsliq_n_s32",
+                "vsli_n_s64",
```

Comment:
I seem to have to add these for the `verify_all_signatures` tests to pass, I'm guessing because I've put the shift-and-insert tests in their own file (mimicking what's done for the table lookup tests, which also appear in this list of signatures to skip).

Reply:
I believe that they are not being detected because you are generating the tests within a macro.